frontend/src/scenes/setup/PreflightCheck.js

```diff
@@ -0,0 +1,236 @@
+                        justifyContent: 'center',
+                        marginLeft: 32,
+                        flexDirection: 'column',
+                        paddingTop: 32,
+                    }}
+                >
+                    <Card style={{ width: 600, width: '100%' }}>
+                        <Row style={{ marginBottom: 16 }}>
+                            <b style={{ fontSize: 16 }}>
+                                Preflight check{' '}
+                                {state.mode && (
+                                    <span>
+                                        (
+                                        <span
+                                            style={{ textDecoration: 'underline', cursor: 'pointer' }}
+                                            onClick={() => handleModeChange(null)}
+                                        >
+                                            {state.mode}
+                                        </span>
+                                        )
+                                    </span>
+                                )}
+                            </b>
+                            <Button
+                                type="default"
+                                style={{ position: 'absolute', right: 16 }}
+                                data-attr="preflight-refresh"
+                                icon={<SyncOutlined />}
+                                onClick={() => window.location.reload()}
+                            >
+                                Refresh
+                            </Button>
+                        </Row>
+                        <div>
+                            Tell us what you plan to do with this installation to make sure your infrastructure is ready
+                        </div>
+                        <div
+                            className="text-center"
+                            style={{ padding: 32, display: 'flex', justifyContent: 'center', maxWidth: 600 }}
+                        >
+                            {!state.mode && (
+                                <>
+                                    <Button
+                                        type="default"
+                                        data-attr="preflight-experimentation"
+                                        onClick={() => handleModeChange('Experimentation')}
+                                    >
+                                        🧪 Just experimenting
+                                    </Button>
+                                    <Button
+                                        type="primary"
+                                        style={{ marginLeft: 16 }}
+                                        data-attr="preflight-live"
+                                        onClick={() => handleModeChange('Live')}
+                                    >
+                                        🚀 Live implementation
```

Comment:
[AntD Icons](https://ant.design/components/icon/) would possibly better match the rest of our design.

Reply:
Personally I like the emojis better, but I tend to agree, we don't use them anywhere else. I'll do AntD icons for now. We can revisit this in the future.

<img width="500" src="https://user-images.githubusercontent.com/5864173/90822688-6f95a180-e2fa-11ea-9e40-b9d291d92411.png">